src/selectors/petitionMetrics.js

```diff
@@ -33,7 +34,8 @@ export default (petition) => {
         minimum: 0,
         maximum: requiredVotes,
         value: amountVotes
-      }
+      },
+      votingActive: votingActive
```

Comment:
I was seeing this lying outside the `metric` key, at the base of the object. but it doesn't matter I guess, we can always move it later if another instance (not related to metrics) is needed